workflows/theiacov/wf_theiacov_illumina_pe.wdl

```diff
@@ -439,6 +449,7 @@ workflow theiacov_illumina_pe {
     # QC_Check Results
     String? qc_check = qc_check_task.qc_check
     File? qc_standard = qc_check_task.qc_standard
- 
+    # Capture percentage_mapped_reads from ivar_consensus task
+    Float? percentage_mapped_reads = ivar_consensus.percentage_mapped_reads 
```

Comment:
can you delete line 441 and turn 453 into `select_first([ivar_consensus.percentage_mapped_reads, flu_stats_n_coverage.percentage_mapped_reads])` so that this output only appears once; like how you have for theiacov_ont